generic/tests/netperf.py

```diff
@@ -93,6 +93,7 @@ def env_setup(session, ip, user, port, password):
         error.context("Setup env for %s" % ip)
         ssh_cmd(session, "iptables -F", ignore_status=True)
         ssh_cmd(session, "service iptables stop", ignore_status=True)
+        ssh_cmd(session, "systemctl stop firewalld.service", ignore_status=True)
```

Comment:
Yes, it should be executed on each guest, use like this, we do not care the guest version and which service iptable/firewalled was running on the vm, just make sure that iptable/firewallwd is stopped on vm, and it's very important for the performance.